data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,224 @@ mission "Remnant: Visit to Paradise 1"
+			`	He frowns a bit "It worked, although not perfectly. There are chunks missing, some of which seriously impair its functionality. But it is back in the primary weft, so now we have time to work on it and figure out what happened to it.`
+			choice
+				`	"I think I get the picture. Let me know if I can help."`
+					goto shatteredhelp
+				`	"What is this 'Primary Weft' you have mentioned?"`
+			`	Another Remnant steps forward. "The Weft is what we term the mesh of dimensions, energy, and time that form a subset of the universe. A layer, of sorts." He looks thoughtful. "What you probably know as 'Hyperspace' is what we technically call the secondary weft. There are a few others we are fairly confident exist, and several that we have hypothesized might exist but we have no means of confirming. Practically, each of these overlaps with the primary weft to some extent." His gestures take on an amused tone. "Of course, what we call the Primary Weft is only primary to those of us who live here. For all we know, there may be people living elsewhere that consider their own weft to be the primary one. Nothing says ours is special or the best or anything of the sort."`
+			label shatteredhelp
+			`	"In terms of help, could you go to Caelian? We have sent word ahead, and our engineers are retrieving some of our remaining equipment from the time of the Exodus. We are hoping that it might be of use to help us repair the Shattered Light, or at least understand what happened to it.`
+			`	"Before you leave, though, there was some stuff that survived intact we would like to share with you." Behind her five androids emerge from one of the cargo hold, each one carrying a large trunk of equipment. "These are known as NDR-114 Androids, and were cutting edge innovations back when the exodus occurred. Don't expect too much of them: They are not self-aware, sentient, or even particularly advanced artificial intelligence. They are, however, excellent extensions of a ship's self-maintenance systems. As it turns out, a large number of these androids were instrumental in keeping the Shattered Light flying all these years. We have plenty to study, we thought you might appreciate having a few." She makes a dismissive gesture. "If they end up not being helpful for you, our outfitters will be able to re-assign them to another ship."`
+			scene "outfit/android"
+			`	"Embers light your path, Captain <first>." With that, the crowd disperses.`
+		outfit "NDR-114 Android" 5
+
+
+
+mission "Remnant: Shattered Light 5"
+	name "Brilliant Energy"
+	description "Retrieve equipment from <destination> to help research the Shattered Light."
+	landing
+	source "Viminal"
+	destination "Caelian"
+
+
+
+mission "Remnant: Shattered Light 6"
```

Comment:
Uh... Something seems to have eaten it. That's a problem. Not a huge one, but even so... I'll look into this.

Reply:
Alright, dug into this and re-discovered what's going on. Namely, that all the conversation that starts this mission is part of the conclusion conversation for Shattered Light 4. Conversely, all the complete text is part of the on offer text for SL6.

This mission literally only exists so that there's a map marker and something in the mission list so that player's don't forget where they are supposed to be going.